examples/camera-app/esp32/main/Button.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+ #include "Button.h"
+
+ #define GPIO_INPUT_IO_0 9
+
+ #define GPIO_INPUT_PIN_SEL (1ULL << GPIO_INPUT_IO_0)
+ #define ESP_INTR_FLAG_DEFAULT 0
+
+ static const char TAG[] = "Button";
+
+ static Button::ButtonPressCallback button_press_handler = nullptr;
+
+ static void gpio_isr_handler(void * arg)
+ {
+     if (button_press_handler != nullptr)
+     {
+         button_press_handler();
+     }
+ }
+
+ void Button::Init()
```

Comment:
Are we using any buttons on C6 in this example? If no, please remove this file and the header.